.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,19 @@
+name: Node.js CI
+on:
+  push:
+    branches: [ master ]
+    pull_request:
```

Comment:
Looks like the indentation of this is wrong ?

Reply:
oops. fixed.